### PR TITLE
fix-NXP-26332-disable-save-as

### PR DESCRIPTION
**Pull request by ataillefer**

### CODE REVIEW

**Comment by kevinleturc** [comment]
👍

File: nuxeo-wopi/src/test/java/org/nuxeo/wopi/jaxrs/TestFilesEndpoint.java [hunk 1]
```diff
@@ -820,56 +819,23 @@ public void testPutRelativeFile() throws IOException {
 
         checkPostNotFound(headers);
 
-        // fail - 501 - no headers
+        // fail - 400 - no headers
         try (CloseableClientResponse response = post(johnToken, data, headers, blobDocFileId)) {
             assertEquals(501, response.getStatus());
```
[the remark this answers is not in the document]
Comment (ataillefer) [reply nuxeo-wopi/src/test/java/org/nuxeo/wopi/jaxrs/TestFilesEndpoint.java]: Yes indeed :smile:, fixed.

File: nuxeo-wopi/src/main/java/org/nuxeo/wopi/jaxrs/FilesEndpoint.java [hunk 2]
```diff
@@ -752,13 +730,13 @@ protected void addUserMetadataProperties(Map<String, Serializable> map) {
     }
 
     protected void addUserPermissionsProperties(Map<String, Serializable> map) {
-        boolean hasAddChildren = session.exists(doc.getParentRef())
-                && session.hasPermission(doc.getParentRef(), SecurityConstants.ADD_CHILDREN);
         boolean hasWriteProperties = session.hasPermission(doc.getRef(), SecurityConstants.WRITE_PROPERTIES);
+        boolean canConvert = Framework.getService(org.nuxeo.wopi.WOPIService.class).getActionURL(blob,
```
Comment (troger) [inline nuxeo-wopi/src/main/java/org/nuxeo/wopi/jaxrs/FilesEndpoint.java]: Not sure we need the full import here.

File: nuxeo-wopi/src/test/java/org/nuxeo/wopi/TestWOPIService.java [hunk 3]
```diff
@@ -85,6 +86,9 @@ public void testGetActionURL() {
                 wopiService.getActionURL(blob, "view"));
         assertEquals("https://excel.officeapps-df.live.com/x/_layouts/xlviewerinternal.aspx?edit=1&IsLicensedUser=1&",
                 wopiService.getActionURL(blob, "edit"));
+        blob.setFilename("file.xls");
+        assertEquals("https://excel.officeapps-df.live.com/x/_layouts/ExcelConvertAndEdit.aspx?IsLicensedUser=1&",
+                wopiService.getActionURL(blob, "convert"));
```
Comment (troger) [inline nuxeo-wopi/src/test/java/org/nuxeo/wopi/TestWOPIService.java]: We could use `Constants#ACTION_CONVERT` here (well, same for the above actions, my bad)

**Pull request merged**